docs/hosting/external-storage.md

```diff
@@ -38,6 +38,8 @@ Create and configure a bucket following the [AWS documentation](https://docs.aws
 }
 ```
 
+Set a bucket-level lifecycle configuration so that S3 automatically deletes old binary data. n8n delegates pruning of binary data to S3, so setting a lifecycle configuration is required unless you would like to preserve binary data indefinitely.
```

Comment:
```suggestion
Set a bucket-level lifecycle configuration so that S3 automatically deletes old binary data. n8n delegates pruning of binary data to S3, so setting a lifecycle configuration is required unless you want to preserve binary data indefinitely.
```